src/Features/CSharp/Portable/AddImport/CSharpAddImportFeatureService.cs

```diff
@@ -132,6 +132,10 @@ protected override bool CanAddImport(SyntaxNode node, CancellationToken cancella
         protected override bool CanAddImportForDeconstruct(string diagnosticId, SyntaxNode node)
             => diagnosticId == CS8129;
 
+        protected override bool CanAddImportForGetAwaiter(string diagnosticId, SyntaxNode node) =>
+            diagnosticId == CS1061 &&
+            node.AncestorsAndSelf().Any(n => n is AwaitExpressionSyntax);
```

Comment:
IDE style is => on the next line for methods.  thanks!

Reply:
I should've realized that... there was an example of exactly that on the line above. Haha. Thanks for catching that